shotover/src/runner.rs

```diff
@@ -72,6 +80,7 @@ impl Default for ConfigOpts {
             stack_size: 2097152,
             log_format: LogFormat::Human,
             hotreload_socket: None,
+            hot_reload_gradual_shutdown_seconds: 10,
```

Comment:
this should match the default above

... actually, I'm not sure why we even have this default implementation, it seems unused.
Can you remove the whole `impl Default for ConfigOpts { .. }` block.